packages/react-button/src/components/Button/useButtonClasses.ts

```diff
@@ -162,6 +177,32 @@ export const useButtonClasses = makeClasses<ButtonState>({
     '--button-disabled-borderColor': 'var(--color-brand-disabled-borderColor)',
     '--button-disabled-contentColor': 'var(--color-brand-disabled-contentColor)',
     '--button-disabled-iconColor': 'var(--color-brand-disabled-iconColor)',
+
+    '@media screen and (-ms-high-contrast: active)': {
+      MsHighContrastAdjust: 'none',
```

Comment:
let's use new API `forced-colors` and `forced-color-adjust` [see here](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/forced-colors) instead of the legacy `-ms` ones

Reply:
Done